subscriptions/configuration/subscription_products.rst

```diff
@@ -0,0 +1,44 @@
+=========================
+Add subscription products
+=========================
+
+In order to be able to suggest and sell your products to your prospects and customers, a
+subscription must have its products properly set. As a result, you can create quotations and
+continue the sales flow to track the number of subscriptions you sell and manage the revenue they
+generate.
```

Comment:
Doesn't make sense

Is it the intro to the doc? 

I'd scrap it and add a bit of intro below in the "create your first sub product). Something like, to sell Subscriptions, you'll first have to add your subscription products with the right settings, then you'll be able to manage it just like any other product and follow their sales cycles, the revenue they generate and track how many you sold.

Reply:
The intro about Subscriptions (in general) is mentioned in the first doc "Use subscription templates" :)